PermanentOrg/web-app
# Update checklist when tasks are completed

Right now the checklist functions well, but users need to refresh the page when they complete tasks to seem them checked off. This PR updates the checklist so it automatically refreshes when an item may be checked off. It does this by refreshing whenever an event is sent to the AnalyticsService.

**Steps to test:**
1. Create a new account with the `hideChecklist` field set to true.
2. Complete an action
3. The checklist should update whenever a relevant task is completed.

**Considerations before release**
This behavior was mainly tested on a local environment without any latency. We might want to add a slight delay before refreshing the checklist to account for network latency.
